### DISCUSSION TITLE
blog/v1-33-released/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#347] [Page comments]
# blog/v1-33-released/

Badges for project creators in comments, a keyboard shortcut for pausing, and more!

https://scratchaddons.com/blog/v1-33-released/

**Comment by MaterArc** [comment]
First comment!

**Comment by SpryzenR** [comment]
What are the changes that Scratch is making?

**Reply by CST1229** [reply]
Adding a high-contrast blocks mode to the editor (theme switching), and making most blue things on the website and editor purple.

You can try the editor changes here: https://scratchfoundation.github.io/scratch-gui/beta/

**Reply by SpryzenR** [reply]
Thanks

**Comment by mybearworld** [comment]
Za-Chary's comment there is quite nice

**Reply by Secret-chest** [reply]
Where?

**Reply by mybearworld** [reply]
In the screenshot of the project creator addon "A lot of 2d platformers on Scratch are still technically 3d"
